src/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -1784,16 +1782,19 @@ public abstract partial class DbConnection : System.ComponentModel.Component, Sy
     {
         protected DbConnection() { }
         [System.ComponentModel.DefaultValueAttribute("")]
-        [System.ComponentModel.RefreshPropertiesAttribute((System.ComponentModel.RefreshProperties)1)]
+#pragma warning disable 0618
+        [System.ComponentModel.RecommendedAsConfigurableAttribute(true)]
+#pragma warning restore 0618
+        [System.ComponentModel.RefreshPropertiesAttribute((System.ComponentModel.RefreshProperties)(1))]
         [System.ComponentModel.SettingsBindableAttribute(true)]
         public abstract string ConnectionString { get; set; }
         public virtual int ConnectionTimeout { get { throw null; } }
         public abstract string Database { get; }
         public abstract string DataSource { get; }
         protected virtual System.Data.Common.DbProviderFactory DbProviderFactory { get { throw null; } }
-        [System.ComponentModel.Browsable(false)]
+        [System.ComponentModel.BrowsableAttribute(false)]
         public abstract string ServerVersion { get; }
-        [System.ComponentModel.Browsable(false)]
+        [System.ComponentModel.BrowsableAttribute(false)]
         public abstract System.Data.ConnectionState State { get; }
         public virtual event System.Data.StateChangeEventHandler StateChange { add { } remove { } }
         protected abstract System.Data.Common.DbTransaction BeginDbTransaction(System.Data.IsolationLevel isolationLevel);
```

Comment:
What about EnlistTransaction which was removed from netstandard to break the dependency on Transactions. Do you want to keep that in the .NET Core version?